Cargo.lock

```diff
@@ -171,48 +171,46 @@ dependencies = [
  "byteorder",
  "digest",
  "lazy_static",
- "libflate",
+ "libflate 1.4.0",
  "log",
  "num-bigint",
  "quad-rand",
  "rand",
  "regex",
  "serde",
  "serde_json",
- "strum 0.25.0",
- "strum_macros 0.25.2",
+ "strum",
+ "strum_macros",
  "thiserror",
- "typed-builder",
+ "typed-builder 0.14.0",
  "uuid",
  "zerocopy",
 ]
 
 [[package]]
 name = "apache-avro"
```

Comment:
Why do we have two `apache-avro` dependencies? Is this expected?

Reply:
More discussion on having 2 (and even possibly 3) versions of **arrow** (not avro): https://github.com/risingwavelabs/risingwave/pull/13600#discussion_r1413470796